actions/changelog-generate/action.yaml

```diff
@@ -74,16 +73,11 @@ runs:
           fi
         fi
         echo "fromtag=$setfrom">> $GITHUB_OUTPUT
-        if [ -n "${{ inputs.config }}" ]; then
-          path="$GITHUB_ACTION_PATH/changelog-config.json)"
+        if [ -z "${{ inputs.config }}" ]; then
+          path="$GITHUB_ACTION_PATH/changelog-config.json"
         else
           path=${{ inputs.config }}
         fi
-        if [ ! -f "$path" ]; then
-          echo "changelog config file $path doesn't exist"
-          exit 1
-        fi
-        echo "path=$path" >> "$GITHUB_OUTPUT"
```

Comment:
why did you remove this check and how does the `path` output variable work now?

Reply:
the check was removed because it was not working as expected. the path works basically the same.. if a config was provided then it will be used; else the one from the action will be used

Reply:
what if a config path was provided but there's a mistake and the file doesn't actually exist?
also I believe you still need this last line, otherwise the output variable is not created

Reply:
oh good catch...it should be fine..now..also test in PG